app/views/manageAgent/areYouSure.scala.html

```diff
@@ -21,7 +21,14 @@
 
 @(agentName: String, agentCode: Long, chosenListYear: String, backLink: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
-@invalidListYear = @{if (chosenListYear.equals("2023")) "2017" else "2023"}
+@invalidListYear = @{chosenListYear match {
+        case "2026" => messages("ratingsListConfirmation.theyWillNot", "2023", "2017")
+        case "2023" => messages("ratingsListConfirmation.theyWillNot", "2026", "2017")
+        case "2017" => messages("ratingsListConfirmation.theyWillNot", "2026", "2023")
+        case _ => ""
+    }
+}
```

Comment:
The AreYouSureController has this check that needs to be removed, as otherwise chosenListYear will never be 2026. I think I missed this if statement when removing the old flagged stuff

```scala
      if (chosenListYear == "2017" || chosenListYear == "2023")
```